fenix/app/src/main/java/org/mozilla/fenix/translations/TranslationsDialogBottomSheet.kt

```diff
@@ -67,45 +67,29 @@ private val ICON_SIZE = 24.dp
 /**
  * Firefox Translations bottom sheet dialog.
  *
+ * @param translationsDialogState The current state of the Translations bottom sheet dialog.
  * @param learnMoreUrl The learn more link for translations website.
- * @param showFirstTimeTranslation Whether translations first flow should be shown.
- * @param translateFromLanguages Translation menu items to be shown in the translate from dropdown.
- * @param translateToLanguages Translation menu items are to be shown in the translate to dropdown.
- * @param initialFrom The initial selection for the translate from dropdown.
- * @param initialTo The initial selection for the translate to dropdown.
- * @param translationError The type of translation errors that can occur.
+ * @param showFirstTime Whether translations first flow should be shown.
```

Comment:
Could we use a better name here? Does `showFirstTimeTranslation` no longer apply here? What about `showFirstTimeFlow`?

Reply:
Done